hack/version.sh

```diff
@@ -100,7 +100,7 @@ function kubevirt::version::ldflag() {
 function kubevirt::version::ldflags() {
     kubevirt::version::get_version_vars
 
-    SOURCE_DATE_EPOCH=$(git show -s --format=format:%ct HEAD)
+    set +e SOURCE_DATE_EPOCH=$(git show -s --format=format:%ct HEAD)
```

Comment:
Can you instead do `SOURCE_DATE_EPOCH=${KUBEVIRT_SOURCE_DATE_EPOCH-$(git show -s --format=format:%ct HEAD)}` and add `KUBEVIRT_SOURCE_DATA_EPOCH` to the three env variables on top?